index.html

```diff
@@ -27,6 +27,8 @@
     <link rel="apple-touch-icon" sizes="180x180" href="favicon_package\apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="favicon_package\favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="favicon_package\favicon-16x16.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.11.0/devicon.min.css">
```

Comment:
Don't include additional icon packs kindly remove them and use the existing packs only.